packages/pytest-simcore/src/pytest_simcore/docker_compose.py

```diff
@@ -205,27 +227,22 @@ def ops_docker_compose_file(
     return docker_compose_path
 
 
+@pytest.hookimpl()
+def pytest_exception_interact(node, call, report):
+    # get the node root dir (guaranteed to exist)
+    root_directory: Path = Path(node.config.rootdir)
+    failed_test_directory = root_directory / "test_failures" / node.name
+    save_docker_infos(failed_test_directory)

```

Comment:
that is nice! so this hook gets called whenever a test fails?

Reply:
@sanderegg , i think you added that some 9 months ago :-)